Python/Setup/signlayout.proj

```diff
@@ -124,8 +124,6 @@
 
   <Target Name="_UpdateLayoutWithSignedBinaries"
           DependsOnTargets="_GetBinariesInLayout"
```

Comment:
What are those removals? They are still in master.

Reply:
Yeah, we should remove them from master as well. Just haven't ported it that far forward yet.

Removing these means that when the signed files come back with a modified date earlier than the originals (which happened to me yesterday), we will still overwrite the originals. Otherwise, we end up with unsigned files in the bundle.